### ISSUE TITLE
Pdf of examples

### ISSUE THREAD

**Issue by lousapetr** [bcbnz/pylabels#11]
Your work looks very nice, however, I would appreciate if I could see some label sheets in the project. Ideally, all the demos could have corresponding pdfs attached to see the results.

**Comment by fabyc** [comment]
+1

**Comment by hydrargyrum** [comment]
Agreed, without a demo, it's hard to understand the purpose of the project and to evaluate if it fits one's needs.
